tools/mo/requirements_caffe.txt

```diff
@@ -4,4 +4,4 @@ numpy>=1.16.6,<1.25.0
 protobuf>=3.18.1,<4.0.0
 defusedxml>=0.7.1
 requests>=2.25.1
-fastjsonschema~=2.15.1
+fastjsonschema~=2.16.2
```

Comment:
should not we have a range like >2.14,<2.17?

Reply:
more reliable, agree

Reply:
@ilya-lavrenov, can you commit your proposal into this PR please?

Reply:
tests have passed